src/components/Upload.jsx

```diff
@@ -49,6 +56,14 @@ const Upload = ({ setCurrentScript, titles, setTitles }) => {
       .catch((err) => {
         if (err.message === 'script already exists') {
           setUploadMessage({ message: 'That script title already exists.', error: true });
+        } else if (err.message === 'potential file name error') {
+          setUploadMessage({
+            message:
+              "File was not uploaded. Make sure the file name doesn't start with an underscore and only uses numbers, letters, and these symbols: '   .   _   -  ",
+            error: true,
+          });
+        } else if (err.message === 'Could not find a title, potentially invalid file type') {
```

Comment:
You're probably already starting to see the problem with this pattern.  
If error messages are identified by their string content, then you have to have the entire, exact, message duplicated in each place that the error is handled.  

Whenever a message is altered, it risks breaking the error handling that exists elsewhere unless the developer is diligent and searches the codebase for the old message.  
This is generally why error codes exist, since there isn't as much reason for them to change.